firebase.json

```diff
@@ -1,41 +1,19 @@
 {
   "hosting": {
     "public": "build",
-    "headers": [
-      {
-        "source": "apple-app-site-association",
-        "headers": [
-          {
-            "key": "Content-Type",
-            "value": "application/json"
-          }
-        ]
-      }
-    ],
-    "appAssociation": "NONE",
     "ignore": [
       "firebase.json",
       "**/.*",
-      "**/node_modules/**",
-      "src/*"
+      "**/node_modules/**"
     ],
     "rewrites": [
-      {
-        "source": "/privacy",
-        "destination": "/privacy.html"
-      },
-      {
-        "source": "/terms",
-        "destination": "/terms.html"
-      },
-      {
-        "source": "/faq",
-        "destination": "/faq.html"
-      },
```

Comment:
If you remove this, we won't be able to access those pages. The apps are using this urls.